### PR TITLE
Improvements for mobile monitoring docs

### PR DESCRIPTION
**Pull request by pizzaz93**
#5997

### CODE REVIEW

**Comment by x8a** [comment]
Hey, @zuluecho9! This is related to an issue you opened. Would you mind looking at it? Thanks!

**Comment by pizzaz93** [comment]
**Note:** The file name is changed in the navigation .yml we should decide if we're changing the file name before merging.

**Review by zuluecho9 — APPROVED** [review]

**Comment by zuluecho9** [comment]
@pizzaz93 no, I think the change to 'summary' makes sense for the nav file. Unless i'm missing something there.

**Comment by pizzaz93** [comment]
The filename is still overview so it wouldn't link to the page currently. I can finish up that part in renaming the file (and jp) and adding redirects etc.

**Comment by zuluecho9** [comment]
oh sorry @pizzaz93 i see what you mean now. I can do that. you've done enough. take some time off ;)

**Comment by pizzaz93** [comment]
Many thanks 🙂, leaving it in your hands.

**Comment by pizzaz93** [comment]
Potential additional cleanup item, page references: https://github.com/newrelic/docs-website/search?q=mobile-apps-overview-page

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-summary-page.mdx
Comment (pizzaz93) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-summary-page.mdx:97]:
```suggestion
## View the Summary page [#summary-page]
```

**Comment by zuluecho9** [comment]
thank you @pizzaz93 i'll look for other references. (as you maybe noticed, we are having some technical issues where it's not passing checks, hence the wait)

**Pull request merged**
